modules/network/security_groups.tf

```diff
@@ -25,8 +25,8 @@ resource "aws_vpc_security_group_egress_rule" "server_sg_egress_rule" {
 }
 
 resource "aws_security_group" "scheduler_sg" {
-  name        = "scheduler_sg_${var.suffix}"
-  description = "The security group for the Spacelift cron scheduler service."
```

Comment:
The description change compared to v2 and v3 triggered a resource recreation.... I found it easier to just match the description with whatever we have in the V2 Cloudformation template.